### DISCUSSION TITLE
Is there have Invalidate() for the WriteableBitmap?

### DISCUSSION THREAD

**Discussion by dxh-sc** [AvaloniaUI/Avalonia#12914] [General]
i see windows UI have Invalidate method for WriteableBitmap.

**Invalidate() | Requests a draw or redraw of the entire bitmap.**

do you think we can have this to update the WriteableBitmap content?
or has other way to high-performance(with high fps) update the content?

my use case is to update video frame with 60 fps.

**Comment by maxkatz6** [comment]
> Invalidate() | Requests a draw or redraw of the entire bitmap.

WriteableBitmap.Invalidate is planned, but you can call `image.InvalidateVisual()` now, where Image has writeable bitmap as a source.
